docs/action-type-template.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[<ACTION-TYPE>-action-type]]
+=== <ACTION-TYPE> action
+
```

Comment:
What about adding 

++++
<titleabbrev><ACTION_TYPE></titleabbrev>
++++

This would clean up the TOC so that the word action doesn't appear in every title:

**Before**

<img width="281" alt="Screen Shot 2021-02-27 at 10 56 02 AM" src="https://user-images.githubusercontent.com/33642766/109397208-7c6b0180-78ea-11eb-80c9-c74f4da023c3.png">

**After**

<img width="324" alt="Screen Shot 2021-02-27 at 10 59 33 AM" src="https://user-images.githubusercontent.com/33642766/109397287-e84d6a00-78ea-11eb-8e56-d3a053615ef6.png">

Reply:
Updated.